__tests__/app.test.js

```diff
@@ -68,14 +68,6 @@ describe("/api/articles/:article_id", () => {
           expect(res.body.msg).toBe("bad request");
         });
     });
-    test("404 - given non existent id with no data, returns message: not found", () => {
```

Comment:
Would say this is a valid test (unless someone has said otherwise) - how come we've taken it out?

Reply:
someone said it was handled globaly

Reply:
for all 404's